packages/graphrag-storage/graphrag_storage/storage_config.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024 Microsoft Corporation.
+# Licensed under the MIT License
+
+"""Storage configuration model."""
+
+from pydantic import BaseModel, ConfigDict, Field
+
+
+class StorageConfig(BaseModel):
+    """The default configuration section for storage."""
+
+    model_config = ConfigDict(extra="allow")
```

Comment:
This doesn't seem like the right name for this, and maybe should be punted to a broader discussion about how we want to handle open-ended config

Reply:
Not sure what else to call it. Got any suggestions?